rclcpp/include/rclcpp/node_interfaces/node_parameters_interface.hpp

```diff
@@ -30,6 +30,7 @@
 
 namespace rclcpp
 {
+
```

Comment:
nitpick: seems like an odd change for this pr

Reply:
fixed in https://github.com/ros2/rclcpp/pull/1532/commits/124c948f28ecd09802900663c56dfe25f9370f6e